app/main.py

```diff
@@ -1 +1,66 @@
-# write your code here
+class SoftwareEngineer:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.extend([skill])
```

Comment:
if you're adding only 1 element, there is a better `append` method

Reply:
In a checklist there was exactly an extend, but - ok) Fixed.